core/pom.xml

```diff
@@ -140,45 +127,34 @@
         <!-- Lucene spatial -->
 
 
-        <!-- START: dependencies that are shaded -->
+        <!-- START: dependencies that might be shaded -->
```

Comment:
I think this comment is not useful anymore if not all depds under it are shaded :)